changelog/bug-1578900.md

```diff
@@ -0,0 +1,22 @@
+level: major
+reference: bug 1578900
+---
+
+* Worker Manager AWS Provider now requires the `ec2:DescribeRegions` permission in addition to the previous permissions.
+  The full permissions set is documented in the  deploying workers section of the manual.
+* Worker pools that use the Worker Manager AWS Provider must now provide an array of `launchConfigs` rather than a single
```

Comment:
It's always been the array of `launchConfigs`, the change is that we used to select one of them, but now we use them all